apps/recovery-portal/src/main/webapp/error.jsp

```diff
@@ -44,13 +46,24 @@
         } else if (StringUtils.isNotBlank(request.getParameter("tenantDomain"))){
             tenantDomain = request.getParameter("tenantDomain").trim();
         }
+
+        Boolean validCallBackURL = false;
```

Comment:
Wouldn't it be better if we can prefix `is` to boolean variables? For example,

```suggestion
        Boolean isValidCallBackURL = false;
```

Reply:
Yes. It is better. Fixed by - https://github.com/wso2/identity-apps/pull/3891/commits/c567dffe87086a9649e061686c12c755e7350637

Reply:
Thanks bro